src/components/REPLInput.js

```diff
@@ -33,10 +40,22 @@ export default class REPLInput extends React.Component {
           {'> '}
         </span>
         <input style={styles.input}
+               type="text"
                value={this.props.code}
                onChange={event => this.props.changeREPLCode(event.target.value)}
                onKeyPress={event =>
-                 event.key === "Enter" && this.props.onRun(this.props.code)} />
+                 event.key === "Enter" && this.props.onRun(this.props.code)}
+               onKeyDown={event => {
+                 if (event.key === "ArrowUp") {
+                   this.props.prevREPLCode();
+                   this.props.displayNewREPLHistoryCode();
+                 } else {
+                   if (event.key === "ArrowDown") {
+                     this.props.nextREPLCode();
+                     this.props.displayNewREPLHistoryCode();
```

Comment:
It seems strange that when you use the arrow keys, two separate actions have to get dispatched instead of just one. Why not just have `prevREPLCode` and `nextREPLCode` and have the associated reducers do whatever the `displayNewREPLHistoryCode` reducer does?